libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md

```diff
@@ -0,0 +1,97 @@
+# iFlight 2RAW H743 Flight Controller
+
+The iFlight 2RAW H743 is a flight controller produced by [iFlight](https://shop.iflight.com/Thunder-H7-Flight-Controller-Pro2200).
+
+## Features
+
+ - MCU - STM32H743 32-bit processor running at 480 MHz
+ - IMU - ICM42688P
+ - Barometer - SPL06-001
+ - OSD - AT7456E
+ - Onboard Flash: 1GBit (not useable by ArduPilot)
```

Comment:
why not? shouldnt we enable it (even requiring creating a driver before merging) since its the only logging means?

Reply:
No driver. Vendor wants to provide the board without logging support

Reply:
Will be boldly noted in the wiki